e2e/fragments/dq/witnesses.page.js

```diff
@@ -28,14 +40,27 @@ module.exports = {
       I.click(this.fields(party).witnessesToAppear.options.yes);
     });
 
-    await this.addWitness(party);
+    let isHNLEnabled = await checkToggleEnabled('hearing-and-listing-sdo');
+    if (!isHNLEnabled || !(['preview', 'demo'].includes(config.runningEnv))) {
```

Comment:
I think we want to be looking at the old fields in preview and demo too since we exclude them in these environments. Maybe we just need to rely on the feature toggle only?